best-practices/development-tools/azure/devops/deployment/azure-app-functions/azure-app-functions-deployment.md

```diff
@@ -0,0 +1,26 @@
+# Deployment using Azure DevOps
+
+## Pre-Deployment
+### Install AzureDevOps Extensions
+There are two extensions should be installed before in AzureDevOps [ReplaceTokens](https://marketplace.visualstudio.com/items?itemName=citrus-andriessen.cag-replace-tokens&targetId=af056fb1-e0c2-4678-9ac3-dee687452af2&utm_source=vstsproduct&utm_medium=ExtHubManageList).
+
+## .NET Core with deployment to Azure Function Apps
+### Option 1
+1. Go to the Pipelines page in Azure DevOps and choose New Pipeline.
+2. Choose `Use the classic editor`.
+3. Select GitHub as a source. (You might need to authorize Azure Pipelines with GitHub via OAuth. If so, choose "Authorize using OAuth ". This will open a new window to either log in with GitHub or grant permission to authorize Azure Pipelines with the logged in GitHub account. Granting access to the team organization where the repository is located might require an additional step: approval by an owner from that organization.) Select the target Repository and the default github branch then continue.
+4. On the Choose a Template page, select `ASP.NET Core` template and choose `Apply`.
+5. Choose `Triggers` tab from top and make sure to check `Enable continuous integration` under Continuous integration. (To trigger deployment process after each commit to the default github branch).
+6. Click on `Save`.
+
+### Option 2
+Create a new Pipeline then choose Github `yaml`. Choose the repository from GitHub, then choose `ASP.NET Core` template, then attach to `azure-pipelines-azure-app-functions-cd.yml` template.
```

Comment:
We may want to specify what we are connecting to in regards to Github yaml.

Reply:
Actually this is an option in Azure DevOps called `Github yaml` I'll try to merge these 2 words.